admin_guide/managing_networking.adoc

```diff
@@ -1322,3 +1322,43 @@ such as Google can construct a list of sites that have `preload` set. Browsers
 can then use these lists to determine which sites to only talk to over HTTPS,
 even before they have interacted with the site. Without `preload` set, they need
 to have talked to the site over HTTPS to get the header.
+
+[[admin-guide-enabling-iperf]]
+== Troubleshooting Throughput Issues
+
+There are situations where the applications deployed through {product-title} can cause 
+network throughput issues such as unusually high latency between specific services. 
+
+Use the following steps to analyze performance issues if pod logs do not reveal any cause of the problem:
```

Comment:
I'd make reviewing the logs the first step. Do you have any suggestions for what the logs might reveal?
maybe "methods" instead of "steps" or convert the list to ordered instead of unordered.

Reply:
@kalexand-rh The BZ specifically called out for steps when "the pod logs don't reveal any cause of the problem." But, good thought.
@jeremyeder @mffiedler @pecameron Are there any entries in the logs (journalctl?) that might indicate latency and/or throughput issues?